### DISCUSSION TITLE
Installing RabbitMQ in WIndows Server 2012

### DISCUSSION THREAD

**Discussion by filbertumbawa15** [rabbitmq/rabbitmq-server#11183] [Questions] [answered]
Hello, I wanna ask how to installing RabbitMQ in windows Server 2012?
![a](https://github.com/rabbitmq/rabbitmq-server/assets/77656601/db8f5d43-1f9b-46c7-bb86-70795ff2a492)
When i'm started, i'm got an error like this, and it can't started. But, when i'm installing in WIndows 10, it solved, not have an issue. 

You've ever got a problem like that?
Thanks

**Comment by michaelklishin** [comment, marked as answer]
No one can suggest what may be going on from a single screenshot.

In any case, Windows Server 2012 has [reached end of life](https://learn.microsoft.com/en-us/lifecycle/announcements/windows-server-2012-r2-end-of-support). 

It is not a supported distribution, our team does not do any testing on it (it's been the case for several years now).

Consider using a modern version of Windows and [Chocolatey packages](https://www.rabbitmq.com/docs/install-windows#chocolatey).
